contracts/AccessTokenVerifier.sol

```diff
@@ -60,29 +60,28 @@ contract AccessTokenVerifier is IAccessTokenVerifier, KeyInfrastructure {
         return keccak256(abi.encode(TOKEN_TYPEHASH, token.expiry, hash(token.functionCall)));
     }
 
+    function verifySignerOf(
+        AccessToken calldata token,
+        uint8 v,
+        bytes32 r,
+        bytes32 s,
+        address expectedSigner
+    ) external view returns (address) {
+        address signer = _retrieveSignerFromToken(token, v, r, s);
```

Comment:
I think it's enough to just do `return _retrieveSignerFromToken(token, v, r, s);` here